README.md

```diff
@@ -25,11 +25,11 @@ For either platform, you can alternatively pass down a child element of your cho
 npm install react-native-picker-select
 
 # React Native users
-npm install @react-native-picker/picker
+npm install react-native-picker-select
```

Comment:
That's is not an typo. It's intended because this module uses `@react-native-picker/picker` as an dependency.